src/main/fc/rc_adjustments.c

```diff
@@ -794,6 +806,7 @@ void processRcAdjustments(controlRateConfig_t *controlRateConfig)
 #ifdef USE_OSD_PROFILES
             && adjustmentState->config->adjustmentFunction != ADJUSTMENT_OSD_PROFILE
 #endif
+            && adjustmentState->config->adjustmentFunction != ADJUSTMENT_LED_PROFILE
```

Comment:
there is a USE_OSD_PROFILES guard bot not one for USE_LED_PROFILES.

the changes to the switch case should be reverted and a #ifdef for led profiles should be added here or the #ifdef for osd profiles should be removed here

Reply:
Done.  (option 1)